scripts/ci_hardware_dcap_tests.sh

```diff
@@ -11,7 +11,6 @@ teardownAESM
 startAESMContainer
 
 # Run the DCAP attestation tests in Debug and Simulation modes using integration tests.
-# Note, the DCAP run doesn't test the samples as they can be run on the EPID machines which are cheaper.
```

Comment:
This script, and any related to it, are no longer called and can be deleted. Since we moved to TC Cloud, all our hardware (i.e. debug mode) tests are done in DCAP since all the agents are in Azure.